### ISSUE TITLE
DefaultWordRenderer no longer allows fonts to be set

### ISSUE THREAD

**Issue by bluemods** [logicsquad/nanocaptcha#21]
For my use case I prefer to set my own fonts for this renderer, but with 2.0, this can only be done via reflection like so:

```java

    private static final DefaultWordRenderer RENDERER = makeWordRenderer();

    private static DefaultWordRenderer makeWordRenderer() {
        try {
            DefaultWordRenderer.Builder builder = new DefaultWordRenderer.Builder();
            builder.color(Color.WHITE);
            DefaultWordRenderer renderer = builder.build();

            Field field = renderer.getClass().getDeclaredField("fonts");
            field.setAccessible(true);

            @SuppressWarnings("unchecked")
            List<Font> fonts = (List<Font>) field.get(renderer);
            fonts.clear();
            fonts.add(new Font("Arial", Font.BOLD, 40));
            return renderer;
        } catch (Exception e) {
            throw new RuntimeException(e);
        }
    }
```

I would prefer not to do it this way and to allow me to set the fonts in the Builder class, but it's no big deal. I appreciate this repo.

**Comment by paulhoadley** [comment]
We should be able to easily add support for custom fonts to `DefaultWordRenderer.Builder`. There was some pruning of deprecated corners of the API in 2.0—my thinking was that we'd prune right back and then add in any requested functionality based on real use cases.

Just out of interest, how were you doing this pre-2.0?

**Comment by bluemods** [comment]
Used the deprecated constructor [here](https://github.com/logicsquad/nanocaptcha/compare/release/1.5...release/2.0#diff-6f62d73f2b0d8feaecc75a23901c9a8a286119da1e3ba8c0f07f353b2848eb70L55)

**Comment by paulhoadley** [comment]
Ah, this one: `DefaultWordRenderer(List<Color> colors, List<Font> fonts)`. Missed it.

**Comment by paulhoadley** [comment]
I've just pushed a solution to `develop` that allows you to set `Font`s using the `Builder`, analogous to the recently added support for `Color`. If you can build the project yourself, you can test this out as 2.1-SNAPSHOT, otherwise I'll be releasing 2.1 very soon (to fix #22).
